content/blog/short-lived-access-tokens/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Announcing Short Lived Access Tokens in Pulumi Cloud"
+
+date: 2024-06-25T09:08:31-07:00
+
+draft: false
+
+meta_desc: Introducing short-lived access tokens in Pulumi Cloud for enhanced security and flexibility, now available in the console and REST API.
+
+meta_image: meta.png
+
+authors:
+    - meagan-cojocar
+
+tags:
+    - releases
+
+---
+
+We are excited to introduce a new feature that our users have been eagerly awaiting: Short-Lived Access Tokens! This enhancement allows you to set an optional expiry date on Personal Access Token, Team Access Token, and Organization Access Token, making them automatically invalid after a specified date. This feature is now available in the [Pulumi Cloud console](https://app.pulumi.com) and the [Pulumi Cloud REST API](/blog/short-lived-access-tokens/create-token.mp4), providing enhanced security and control over your access tokens.
```

Comment:
would be cool if we have doc links for each type of token. I'm sure folks like me would ask what a personal access token is and how they can provision it :)